.github/workflows/prettier.yml

```diff
@@ -0,0 +1,26 @@
+name: Check code format
+
+# This action works with pull requests and pushes
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
```

Comment:
i think this `with` block is not mandatory.
`actions/checkout@v3` should arealy checkout on top of the PR branch :thinking: 

the best example i have for this is the [`std-lib-and-python-virtualenv` job of the `nushell` CI](https://github.com/nushell/nushell/blob/main/.github/workflows/ci.yml#L85), 'cause i partially wrote it myself :eyes:

Reply:
mm okok :+1: